app/views/layouts/application.html.erb

```diff
@@ -31,6 +31,7 @@
     <%- end %>
 
     <%= render_google_universal_analytics_code %>
+    <%= tag :link, rel: 'manifest', href: manifest_path %>
```

Comment:
might as well be just `<link rel='manifest' href="<%= Discourse.base_uri %>/manifest.json">`